Sources/StreamChatUI/Common Views/СhatAvatarView.swift

```diff
@@ -4,7 +4,7 @@
 
 import UIKit
 
-open class AvatarView: View {
+open class СhatAvatarView: View {
```

Comment:
Was there "technically" a reason for changing this? 🤔

Reply:
Please refer to PR description, I explained why some types obtained `Chat` prefix

Reply:
to me the reason for renaming `AvatarView` -> `ChatAvatarView` is the same as our decision to have `CurrentChatUser` instead of `CurrentUser`